strings/ambition_objectives_generic.json

```
@@ -0,0 +1,381 @@
+		"Я стою намного больше, чем мне платят! Нужно добиться повышения зарплаты.",
+		"Как же мне мало платят. Какая несправедливость!",
+		"Почему мне так мало платят? Я хочу больше!",
+		"Почему моя зарплата такая маленькая?",
+		"А у меня довольно большая зарплата. Думаю я могу потратить лишние средства.",
+		"Мне одиноко... Нужно найти друзей...",
+		"Меня гнетёт одиночество... Хочу друга...",
+		"Нужно найти друга...",
+		"А смогу ли я подружиться с [random_crew]?",
+		"[random_crew] выглядит как неплохой друг. Может получится подружиться с ним?",
+		"Я хотел бы подружиться с [random_crew].",
+		"Надо присмотреться к [random_crew], может удастся с ним найти общий язык?",
+		"[random_crew] косо на меня посмотрел... Мне это не нравится...",
+		"[random_crew] какой-то странный. Стоит ли ему помочь?",
+		"Мне не нравится [random_crew]... Что-то с ним не так. Надо выяснить что!",
+		"[random_crew] что-то определенно задумал. Нужно за ним втайне проследить!",
+		"Не хотел бы я столкнуться с [random_crew]...",
+		"Не хотел бы я пересечься с [random_crew]...",
+		"Пора завести новые знакомства. Может с [random_crew]?",
+		"Пора завести себе соперника! [random_crew] подойдет!",
+		"Как же скучно. Пора бы подоставать [random_crew]!",
+		"Нужно объединиться с клоуном и подшутить над [random_crew]!",
+		"А смогу ли я выбесить [random_crew]?",
+		"Надо свести [random_crew] и [random_crew]!",
+		"Почему [random_crew] и [random_crew] так мало знакомы друг с другом? Надо им в этом помочь!",
+		"А смогут ли [random_crew] и [random_crew] полюбить друг друга? Надо проверить.",
+		"Если я смогу заставить пожениться [random_crew] и [random_crew], это будет одна из великих побед в моей жизни!",
+		"Думаю ничего страшного, если [random_crew] и [random_crew] вдруг невзлюбят друг друга? Не без моей помощи...",
+		"Мне кажется или [random_crew] с [random_crew] невзлюбили друг друга? Нужно это исправить!",
+		"Мне скучно. Я думаю [random_crew] развлекёт меня.",
+		"Мне скучно. Думаю не случится ничего страшного, если [random_crew] поучаствует в моих выходках!",
+		"Существует ли любовный треугольник? Врятли... Но может я смогу его устроить между [random_crew], [random_crew] и [random_crew]?",
+		"Мне определенно нужна банда! И я уже заприметил пару её членов: [random_crew], [random_crew], [random_crew] и [random_crew].",
+		"Мне нужна банда! А [random_crew] станет её заместителем!",
+		"Мне нужна банда! Мы будем совершать самые нелегальные вещи на станции! Например украдем всех питомцев! Или хлеб! Или...",
+		"Мне следует познакомиться с [random_crew]!",
+		"Мне следует подружиться с [random_crew]!",
+		"Кажется я влюбился в [random_crew]...",
+		"Я помню как мы когда-то в [random_holiday] встретились с [random_crew]. Надо это повторить!",
+		"Я хочу чтобы [random_crew] и [random_pet] стали лучшими друзьями!",
+		"Я хочу чтобы [random_pet] и [random_pet] стали лучшими питомцами друг для друга!",
```

Животные не могут быть питомцами друг для друга.

```suggestion
		"Я хочу чтобы [random_pet] и [random_pet] стали лучшими друзьями!",
```